roles/ceph-container-common/tasks/fetch_image.yml

```diff
@@ -176,8 +176,14 @@
     - nfs_group_name in group_names
     - ceph_nfs_container_inspect_before_pull.get('rc') == 0
 
+- name: set_fact timeout_command
+  set_fact:
+    timeout_command: "{{ 'timeout -s KILL ' ~ docker_pull_timeout if (docker_pull_timeout != '0') else '' }}"
```

Comment:
What's the default timeout here?

Reply:
`roles/ceph-defaults/defaults/main.yml:558:docker_pull_timeout: "300s"`